pithos/plugins/_mpris.py

```diff
@@ -576,6 +581,17 @@ def CanEditTracks(self):
         '''b Read only Interface MediaPlayer2.TrackList'''
         return False
 
+    @dbus_property(MEDIA_PLAYER2_RATINGS_IFACE, signature='b')
+    def CanRate(self):
```

Comment:
Really though it's on it's own interface named "ExtensionPithosRatings" I don't think anyone's going to have an interface named that? A client wouldn't even see that prop unless they were specifically trying to use that interface.